assets/queries/ansible/aws/ingress_not_restricted/test/positive.yaml

```diff
@@ -0,0 +1,15 @@
+- name: example ec2 group
+  amazon.aws.ec2_group:
+    name: example
+    description: an example EC2 group
+    vpc_id: 12345
+    region: eu-west-1
+    rules:
+      - proto: -1
+        from_port: 0
+        to_port: 0
+        cidr_ip: 0.0.0.0/0
+      - proto: -1
+        from_port: 0
+        to_port: 0
+        cidr_ipv6: ::/0
```

Comment:
add another example so it's easier to distinguish the vulnerabilities with cidr_ip and with cidr_ipv6